docs/design/network-discovery/bfd-port-FSM.puml

```diff
@@ -100,6 +100,8 @@ state ACTIVE {
 ACTIVE --> OFFLINE : offline
 ACTIVE --> INIT_REMOVE : disable
 ACTIVE --> INIT_CLEANUP : kill
+ACTIVE : enter / emit enable bfd-slow-discovery
```

Comment:
no... this is a terrible place to control anything except BFD state. ISL has full knowledge about its current state and used discovery and alive check tools. Only ISL itself must manage these tools.